js/nav.js

```diff
@@ -0,0 +1,46 @@
+const menuIcon = document.querySelector('.menu-icon').firstChild;
+const body = document.querySelector('body'); 
+const siteNav = document.querySelector('.internal-links');
+const extNav = document.querySelector('.external-links');
+const intNav = document.querySelectorAll('.hideable-int-link');
+const mobileNav = document.querySelector('.mobile-nav');
+let menuOpen = false;
+
+// on page load, fill mobile nav with regular nav
+const initMobileNav = ()=> {
+  intNav.forEach(nav => {
+    mobileNav.firstChild.appendChild(nav.cloneNode(true)).className = 'internal-links-mobile'
+  })
+  mobileNav.firstChild.appendChild(extNav.cloneNode(true)).className = 'external-links-mobile'
+}
+initMobileNav();
+
+const toggleMenu = ()=> {
+  // toggle menu icon
+  menuIcon.parentNode.ariaLabel = 
+    (menuOpen)
+    ? 'open'
```

Comment:
Let's do "open menu" and "close menu" instead. It's not clear it's a menu just yet.